modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Sorting.tsx

```diff
@@ -0,0 +1,387 @@
+						{Liferay.Language.get('field')}
+
+						<RequiredMark />
+					</label>
+
+					<DropDown
+						closeOnClick
+						menuWidth="sm"
+						trigger={
+							<ClayButton
+								className="form-control form-control-select"
+								displayType="unstyled"
+							>
+								{field ||
+									Liferay.Language.get('choose-an-option')}
+							</ClayButton>
+						}
+					>
+						<DropDown.ItemList>
+							{fields?.map((field) => (
+								<DropDown.Item
+									key={field}
+									onClick={() => setField(field)}
+								>
+									{field}
+								</DropDown.Item>
+							))}
+						</DropDown.ItemList>
+					</DropDown>
+				</ClayForm.Group>
+
+				<ClayForm.Group>
+					<label htmlFor="sorting">
+						{Liferay.Language.get('sorting')}
+
+						<RequiredMark />
+					</label>
+
+					<DropDown
+						closeOnClick
+						menuWidth="sm"
+						trigger={
+							<ClayButton
+								className="form-control form-control-select"
+								displayType="unstyled"
+							>
+								{sorting.label}
+							</ClayButton>
+						}
+					>
+						<DropDown.ItemList>
+							{SORTING_OPTIONS.map((sortingOption) => (
+								<DropDown.Item
+									key={sortingOption.value}
+									onClick={() => setSorting(sortingOption)}
+								>
+									{sortingOption.label}
+								</DropDown.Item>
+							))}
+						</DropDown.ItemList>
+					</DropDown>
+				</ClayForm.Group>
+			</ClayModal.Body>
+
+			<ClayModal.Footer
+				last={
+					<ClayButton.Group spaced>
+						<ClayButton
+							displayType="secondary"
+							onClick={() => closeModal()}
+						>
+							{Liferay.Language.get('cancel')}
+						</ClayButton>
+
+						<ClayButton
+							disabled={!field}
+							onClick={() => {
+								closeModal();
+
+								onSave();
+							}}
+						>
+							{Liferay.Language.get('save')}
+						</ClayButton>
+					</ClayButton.Group>
+				}
+			/>
+		</div>
+	);
+};
+
+const Sorting = ({fdsView, fdsViewsURL}: FDSViewSectionInterface) => {
+	const [fdsSorting, setFDSSorting] = useState<Array<FDSSortingType> | null>(
+		null
+	);
+
+	const fdsSortingRef = useRef('');
+
+	const getFDSSorting = async () => {
+		const response = await fetch(
+			`${API_URL.FDS_FIELDS}?filter=(${OBJECT_RELATIONSHIP.FDS_VIEW_FDS_FIELD_ID} eq '${fdsView.id}')&nestedFields=${OBJECT_RELATIONSHIP.FDS_VIEW_FDS_FIELD}`
+		);
+
+		if (!response.ok) {
+			openToast({
+				message: Liferay.Language.get(
+					'your-request-failed-to-complete'
+				),
+				type: 'danger',
+			});
+
+			return null;
+		}
+
+		const responseJSON = await response.json();
+
+		const storedFDSSorting = responseJSON?.items;
+
+		if (!storedFDSSorting) {
+			openToast({
+				message: Liferay.Language.get(
+					'your-request-failed-to-complete'
+				),
+				type: 'danger',
+			});
+
+			return null;
+		}
+
+		// @TODO: Get persisted sorting values.
+
+		setFDSSorting([]);
+	};
+
+	const updateFDSFieldsOrder = async () => {
+		const body = {
+			fdsFieldsOrder: fdsSortingRef.current,
+		};
```

Comment:
We can probably inline this value to JSON.stringify instead of creating its own variable